include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -123,7 +123,7 @@ class alignas(sycl::queue) __queue_holder
 inline sycl::queue
 __get_default_queue()
 {
-    static sycl::queue __q(sycl::default_selector_v);
+    static sycl::queue __q;
```

Comment:
1) Which warranties we have that the work will the same as before?
2) May be better to check compiler version?

Reply:
According to the sycl 2020 spec, a default constructed queue ... "Constructs a SYCL queue instance using the device constructed from the default_selector_v.".

Reply:
ok

Reply:
And just to add on: for compilers that do not support `default_selector_v`, similar guarantees are made in older SYCL specifications (SYCL 1.2.1):
```
Constructs a SYCL queue instance using the device returned by an instance of
default_selector
```